pages/common/git-submodule.md

```diff
@@ -9,12 +9,12 @@
 
 - Add a Git repository as a submodule:
 
-`git submodule add {{repository_url}}`
+`git submodule add --branch main {{repository_url}}`
 
 - Add a Git repository as a submodule at the specified directory:
 
-`git submodule add {{repository_url}} {{path/to/directory}}`
+`git submodule add --branch main {{repository_url}} {{path/to/directory}}`
```

Comment:
```suggestion
`git submodule add --branch main {{repository_url}} {{path/to/directory}}`
```
Add `--branch main` as it own example. Keep the first few examples as simple as possible. It'll also give you space to explain why you would do it

Reply:
Can you give an example of these side effects?

Reply:
> Can you give an example of these side effects?

I provide short explanation with link to more deeper explanation in corresponding commit msg, is it clear?

Reply:
Not really. The git message provides an example for why you can do it as an extra action, not why you should not exclude it.

Reply:
@AndreiCherniaev Can you please respond so we can move forwards with the PR

Reply:
How's your availability this month? I'll take over and merge what I see reasonable if you don't have the time.